modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/nhttp/ClientHandler.java

```diff
@@ -158,6 +158,18 @@
     private static final String CONTENT_TYPE = "Content-Type";
 
     /**
+     * Configuration which decides if message size validation is enabled/disabled. If enabled, messages of a size
+     * greater than 'validMaxMessageSize' will be dropped.
+     */
+    private static boolean isMessageSizeValidationEnabled = false;
```

Comment:
Yes, it is true. Thanks for pointing it out. But, we've set the same 2 properties in the passthru transport. Changing this will differ the behaviour of the same property in those 2 configs. WDYT? Should we change this? @djkevincr ?